interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statement in Javascript checks a given statement for a boolean return, and then executes a block of code if the return is true. You can also use else to execute a different block of code if a condition is false.
 
-Researched answer:
+Researched answer: 
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is a software that was created to allow developers to share code freely between eachother. Github is a service that acts as an interface for git, making it easier to use.
 
-Researched answer:
+Researched answer: Git is a version control software that was originally designed to make coordination in coding tasks easier, and has since been used moreso to keep track of changes made to codebases or files in general. Github is a git repository, based on the web and using cloud storage, that acts as an easier way to use git and adds its own user features that make git collaboration better for coders.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The Boolean operators return boolean values. These are the operators like and, or, and not. They are often used to check if a statement is true or false within if/else statements.
 
-Researched answer:
+Researched answer: Boolean operators are also very useful for creating complex logic within code, and are a sort of building block in that sense. They can be used to create complicated decision trees and are present in almost every form of logical code.
```

Comment:
There are also relational operators and you've definitely used them during this assessment.